vector/src/main/java/im/vector/app/features/home/room/list/RoomListFragment.kt

```diff
@@ -95,11 +95,12 @@ class RoomListFragment @Inject constructor(
     )
 
     private val adapterInfosList = mutableListOf<SectionAdapterInfo>()
-    private val concatAdapter = ConcatAdapter()
+    private var concatAdapter : ConcatAdapter? = null
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        setupCreateRoomButton()
+        views.stateView.contentView = views.roomListView
```

Comment:
We do not call `setupCreateRoomButton()` anymore?

Reply:
wow, bad revert. good catch